packages/fast-components-react-base/src/dialog/dialog.spec.tsx

```diff
@@ -29,6 +29,14 @@ describe("dialog", (): void => {
         expect((Dialog as any).name).toBe(Dialog.displayName);
     });
 
+    test("should have correct element type and correct role attribute value", () => {
```

Comment:
Since these are two different DOM nodes (one is the root and one is nested), I would prefer to have these within different test blocks.